core/services/ocr2/plugins/ocr2keeper/evm21/logprovider/provider.go

```diff
@@ -30,6 +30,9 @@ var (
 
 	// AllowedLogsPerUpkeep is the maximum number of logs allowed per upkeep every single call.
 	AllowedLogsPerUpkeep = 5
+
+	readJobQueueSize = 64
+	readLogsTimeout  = 5 * time.Second
```

Comment:
would 5 seconds be enough for upkeeps with a lot of logs?

Reply:
Increasing to 10sec